Lecture2/Task4/com/playtika/ArrayAndLinkedLists.java

```diff
@@ -0,0 +1,130 @@
+package com.playtika;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Random;
+
+public class ArrayAndLinkedLists {
+    public static void main(String[] args) {
+        int a = 0;
+        final Random newRandom = new Random(); // Create Random generator
+
+        List<Integer> createdArrayList = new ArrayList<>();
+        List<Integer> createdLinkedList = new LinkedList<>();
+
+        long x = addRandomtoCreatedArrayListMeasure(newRandom, createdArrayList);
+        System.out.println("Time delta for created ArrayList: " + x);
+
+        long y = addRandomCreatedLinkedListMeasure(newRandom, createdLinkedList);
```

Comment:
It's still a bit unclear for my, why we need to pass a random instance each time?